catalog.html

```diff
@@ -12,11 +12,14 @@
       <p class="main-logo"><a href="index.html"><img src="img/svg/technomart.svg" alt="Техномарт"></a></p>
       <ul>
         <li>
-          <!-- Здесь будет форма -->Поиск</li>
+          <form class="form-search" action="https://echo.htmlacademy.ru" method="post">
+            <input type="search" name="question" placeholder="Найти" id="form-search-question">
+            <label class="visually-hidden" for="form-search-question">Что нужно найти?</label>
```

Comment:
```suggestion
            <label class="visually-hidden" for="form-search-question">Поисковая строка</label>
```
У всех пользователей за время пользования интернета уже сложились определенные стереотипы поведения, которые называются юзер экспириенс (UX). Когда пользователь попадает на поиск, он вводит что искать в поисковой строке, поэтому, по моему мнению, лейбл должен быть "поисковая строка". Можешь не согласиться с моим мнением, тогда обоснуй мне своё.